src/App.tsx

```diff
@@ -1,10 +1,109 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import { USER_ID, getTodos, addNewTodo } from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoStatus } from './types/SortTypes';
+import { ErrorType } from './types/Errors';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { SortButtons } from './components/SortButtons';
+import classNames from 'classnames';
+
+const emptyTodo: Omit<Todo, 'id'> = {
+  completed: false,
+  userId: USER_ID,
+  title: '',
+};
+
+const TodoStatusRoutes: Record<TodoStatus, string> = {
+  [TodoStatus.All]: '/',
+  [TodoStatus.Active]: '/active',
+  [TodoStatus.Completed]: '/completed',
+};
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [newTodo, setNewTodo] = useState<Omit<Todo, 'id'>>(emptyTodo);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [selectedTodoStatus, setSelectedTodoStatus] = useState<TodoStatus>(
+    TodoStatus.All,
+  );
+  const [errorMessage, setErrorMessage] = useState<ErrorType | ''>('');
```

Comment:
if you use a special enum (not just a string) so it makes sense to set null as default value here instead of empty string